builder/openstack/run_config.go

```diff
@@ -35,7 +35,7 @@ type RunConfig struct {
 	SourceImageName string `mapstructure:"source_image_name" required:"true"`
 	// Filters used to populate filter options. Example:
 	//
-	// ``` json {
+	// ```json {
```

Comment:
This one didn't work but I think its because of the `{` that it's in the same line as json
![Screenshot 2020-03-11 at 16 06 57](https://user-images.githubusercontent.com/8526180/76431969-9e563600-63b2-11ea-8d97-13d3c759a08b.png)